src/NServiceBus.AcceptanceTests/Recoverability/When_immediate_retries_fail.cs

```diff
@@ -35,10 +33,7 @@ class Context : ScenarioContext
 
             public int NumberOfTimesInvoked { get; set; }
 
-            public DateTime TimeOfFirstAttempt { get; set; }
-            public DateTime TimeOfSecondAttempt { get; set; }
-
-            public int NumberOfDelayedRetriesPerformed { get; set; }
+            public int NumberOfRetriesAttempted => NumberOfTimesInvoked-1 < 0 ? 0 : NumberOfTimesInvoked-1;
```

Comment:
The formatting of this looks wrong. Shouldn't there be spaces around the `-` signs?